service-catalog/cloud-sql-mysql/README.md

```diff
@@ -17,44 +15,47 @@ The sample highlights a number of Kubernetes and Open Service Broker concepts:
 *   *Binding* the provisioned *service instance* to a Kubernetes application.
 *   Use of the *binding* by the application to access the *service instance*.
 
+The sample application allows users to store and retrieve very simple musician
```

Comment:
statements like "very simple" do not add much and are subjective. Please remove that.